packages/cli/src/commands/storybook.js

```diff
@@ -39,7 +39,7 @@ export const handler = ({ open, port, build, buildDirectory }) => {
   const staticAssetsFolder = path.join(getPaths().web.base, 'public')
   // Create the `MockServiceWorker.js` file
   // https://mswjs.io/docs/cli/init
-  execa(`yarn msw init "${staticAssetsFolder}"`, undefined, {
+  execa(`yarn msw init "${staticAssetsFolder}" --no-save`, undefined, {
```

Comment:
Maybe pass no save instead of undefined in second arg? I remember facing issues passing flags in the Cmd sometimes

Reply:
great point!